Library/Homebrew/cmd/style.rb

```diff
@@ -64,7 +64,7 @@ def check_style_impl(files, output_type, options = {})
     when :json
       json = Utils.popen_read_text("rubocop", "--format", "json", *args)
       # exit status of 1 just means violations were found; others are errors
-      raise "Error while running rubocop" if $?.exitstatus > 1
+      raise "Error while running rubocop" if $?.exitstatus.nil? || $?.exitstatus > 1
```

Comment:
Can you extend the comment above this line to explain why `exitstatus` can be `nil`? I think that's non-obvious and thus worth documenting. Maybe something like “exitstatus can be nil if the RuboCop process crashes, e.g. due to problems with native Ruby extensions”?

While you're here, can you also fix the capitalization of RuboCop (👈 that's how it should be spelled) in the message string?